packages/trader/src/Modules/SmartChartBeta/Components/LaunchModal/__tests__/launch-modal.spec.tsx

```diff
@@ -0,0 +1,78 @@
+import * as React from 'react';
+import { mockStore, StoreProvider } from '@deriv/stores';
+import LaunchModal from '../launch-modal';
+import { render, screen, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import TraderProviders from '../../../../../trader-providers';
+
+const renderLaunchModal = (is_logged_in = true) => {
```

Comment:
I can see that you do not pass args in your tests cases, maybe then it will be possible to remove param?